doc/cascadia/profiles.schema.json

```
@@ -54,6 +54,12 @@
         }
       ]
     },
+    "BellSound": {
+      "default": "#",
```

```suggestion
      "default": "",
```
I don't thing having the default path be `"#"` would be useful haha. Let's just make it the empty string.